modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx

```diff
@@ -215,7 +216,7 @@ export default function EditObjectFolder({
 
 			<Header
 				folder={selectedFolder}
-				hasDraftObjectDefinitions={false}
+				hasDraftObjectDefinitions={elements.some(element => (element as FlowElement<ObjectDefinitionNodeData>).data?.status.code === 2)}
```

Comment:
@andremhirai Add a `?` after `status` like `.data?.status?.code`. I also had an issue while testing the PR because of this.